account-kit/core/src/index.ts

```diff
@@ -31,10 +32,30 @@ export type * from "./schema.js";
 export { AlchemyProviderConfigSchema } from "./schema.js";
 export type { AlchemyProviderConfig } from "./type.js";
 
+// Accounts
```

Comment:
re-exporting these things should make it easier for devs who want to use account-kit core with our accounts. This way you only need one package depending on what you're doing.